docs/interpretation_of_JSON_Schema.md

```diff
@@ -22,7 +22,7 @@ The order of interpretation:
 - `dependencies` only apply to schema dependencies, since property dependencies adds nothing to the underlying model. Any schema dependencies are interpreted and then [merged](#Merging-models) together with the current interpreted model.
 - `enum` is interpreted as is, where each `enum`. Usage of `enum` infers the enumerator value type to the model, but only if the schema does not have `type` specified.
 - `const` interpretation overwrite already interpreted `enum`. Usage of `const` infers the constant value type to the model, but only if the schema does not have `type` specified.
-- [oneOf/anyOf/then/else](#Processing-sub-schemas)
+- [allOf/oneOf/anyOf/then/else](#Processing-sub-schemas)
```

Comment:
You missed:

```suggestion
- [allOf/oneOf/anyOf/if/then/else](#Processing-sub-schemas)
```

or am I wrong? 😄

Reply:
You are, because the `if` does not force any new structure for the schema, so there is no need to process it. `if` is only used to determine if `then` or `else` should be applied to the data 🙂